tests/Cryptor.unit.test.ts

```diff
@@ -1,79 +1,96 @@
-import Cryptor from '../src/Cryptor';
-import * as jest from 'ts-jest';
-import * as crypto from 'crypto';
+import Cryptor from '../src/Cryptor'
+import * as jest from 'ts-jest'
+import * as crypto from 'crypto'
 
 
 describe('Cryptor class', () => {
 	// Define initialization vector for all tests
-	const iv = crypto.randomBytes(16);
+	let initVector: Buffer
+	let key: Buffer
+	let cryptor: Cryptor
```

Comment:
Why did we call this `Cryptor`? That is such a strange name. Can we make it more obvious what this thing is. If it's our crypto utility, why not just name it `Crypto`?

Reply:
well I can't comment on the original naming of `Cryptor`, but `Crypto` sounds more conventional. One issue might be a conflict with the `crypto` library when defining `let crypto: Crypto`.

Reply:
okay so there is no namespace error thrown as we don't import `crypto` in `EncryptedFS.ts`.